src/PSADTree/Commands/GetADTreeGroupMemberCommand.cs

```diff
@@ -31,23 +31,25 @@ protected override void ProcessRecord()
                 return;
             }
 
-            WriteObject(
-                sendToPipeline: Traverse(
-                    groupPrincipal: group,
-                    source: group.DistinguishedName),
-                enumerateCollection: true);
+            _truncatedOutput = false;
```

Comment:
- `bool` is reset on each pipeline input to support a warning per input object.
- Showing the warning first before outputting makes more sense.